spring-cloud-netflix-sidecar/src/main/java/org/springframework/cloud/netflix/sidecar/SidecarMarkerConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.springframework.cloud.netflix.sidecar;
```

Comment:
Needs license header. That is while checkstyle failed in the CI build.

Reply:
added license header.